### PR TITLE
{tools}[GCCcore/13.2.0,GCCcore/13.3.0,system/system] tmux v3.4

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen3
EB_BRANCH=5.0.x

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 4 out of 4 (3 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.12
See https://gist.github.com/bedroge/0d1b6f30e5d7ba5ab399d68427c660a0 for a full test report.

File: easybuild/easyconfigs/t/tmux/tmux-3.4-GCCcore-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+dependencies = [
+    ('libevent', '2.1.12'),
+    ('ncurses', '6.4'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/tmux'],
+    'dirs': []
+}
+
```
Comment (bedroge) [inline easybuild/easyconfigs/t/tmux/tmux-3.4-GCCcore-13.2.0.eb:35]:
```suggestion

sanity_check_commands = ["tmux --help 2>&1 | grep 'usage: tmux'"]

```

File: easybuild/easyconfigs/t/tmux/tmux-3.4-GCCcore-13.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+dependencies = [
+    ('libevent', '2.1.12'),
+    ('ncurses', '6.5'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/tmux'],
+    'dirs': []
+}
+
```
Comment (bedroge) [inline easybuild/easyconfigs/t/tmux/tmux-3.4-GCCcore-13.3.0.eb:35]:
```suggestion

sanity_check_commands = ["tmux --help 2>&1 | grep 'usage: tmux'"]

```

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.12
See https://gist.github.com/bedroge/cb493bd4388ab6fae35f050e27926649 for a full test report.

**Review by bedroge — APPROVED** [review]
Lgtm

**Comment by bedroge** [comment]
Going in, thanks @SebastianAchilles!

**Pull request merged**